src/plugins/felixhayashi/tiddlymap/js/graph/ViewAbstraction.js

```diff
@@ -358,21 +344,20 @@ class ViewAbstraction {
       }
 
       $tm.logger('log', 'Setting config', prop, val);
-      this.config[`config.${prop}`] = val;
+      config[`config.${prop}`] = val;
 
+      // save
+      $tw.wiki.addTiddler(new $tw.Tiddler(
+        utils.getTiddler(this.configTRef),
+        config
+      ));
 
     } else { // not allowed
 
       throw new InvalidArgumentException(...args);
 
     }
 
-    // save
-    $tw.wiki.addTiddler(new $tw.Tiddler(
-      utils.getTiddler(this.configTRef),
-      this.config
-    ));
-
   }
```

Comment:
Moved this up into a narrower scope where `config` is defined. If you follow the code, it looks like having the `addTiddler` at the very end just had it get called extraneously during `setConfig({object})` calls.

Reply:
I agreee, well spotted.